templates/library/.travis.yml

```diff
@@ -1,17 +1,16 @@
-# Use docker-based build environment (instead of openvz)
-sudo: false
```

Comment:
Why would we not use container-based builds? They are faster, and caching won't work without them.